crates/holochain_websocket/src/lib.rs

```diff
@@ -136,7 +136,7 @@ pub async fn connect(
 }
 
 #[derive(Debug, serde::Serialize, serde::Deserialize, SerializedBytes)]
-#[serde(tag = "type")]
+#[serde(rename_all = "snake_case", tag = "type")]
```

Comment:
this is the change which lends consistency to the low level API serialization behaviour. It would be a breaking change for holochain-client-js and other clients

Reply:
This is the doc that provides context around how this surfaced 
https://hackmd.io/wRoLSZRHRniiq_FlMcmHrA

Reply:
Since the other serialization are converting from snake case to camel case, this is consistent.

Reply:
Maybe the client can look for both cases for a while for backward compatibility.

Reply:
Well, it will only be a breaking change for the js client itself, but not for the projects that depend on it